Foundation/Bundle.swift

```diff
@@ -106,6 +114,14 @@ open class Bundle: NSObject {
         _bundle = result
     }
     
+    public convenience init?(_executableURL: URL) {
```

Comment:
@pvieito @millenomi Should this method be public? also `_supportsFHSBundles` and `_supportsFreestandingBundle` above? If its just for testing shouldn't they be `internal`?

Reply:
@spevans If we mark it `internal` we would have to guard the tests with `NS_FOUNDATION_ALLOWS_TESTABLE_IMPORT`, which means that they would not be executed on the Linux CI (and I would like them to be executed on the Linux CI, as I can only test them locally on macOS).

Reply:
I thought `NS_FOUNDATION_ALLOWS_TESTABLE_IMPORT` works for Linux CI because it uses the `debug-foundation` flag. @millenomi Is that not the case?